src/Randomizer.Shared/Models/LogicConfig.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.ComponentModel;
+
+namespace Randomizer.Shared
+{
+    public class LogicConfig
+    {
+        [DisplayName("Prevent Screw Attack Sock Locks"), Description("Requires the morph ball before using screw attacks in locations where you can get stuck"), Category("Logic")]
+        public bool PreventScrewAttackSoftLock { get; set; }
+
+        [DisplayName("Require Ten Power Bombs"), Description("Always require at least two power bomb upgrades for navigation in Super Metroid. Will require tracking multiple power bombs."), Category("Logic")]
+        public bool RequireTwoPowerBombs { get; set; }
+
+        [DisplayName("Fire Rod for Dark Rooms"), Description("Having the fire rod will open up the back of Hyrule Castle, Eastern Palace Armos Knights, and select rooms in Palace of Darkness"), Category("Tricks")]
+        public bool FireRodDarkRooms { get; set; }
+
+        [DisplayName("Infinite Bomb Jump"), Description("Allow using morph bombs to access high locations with specially timed morph bombs"), Category("Tricks")]
```

Comment:
I'd change this from "Allow using" to "You're expected to use" or "You're expected to be able to use"